### ISSUE TITLE
Progress not working with php

### ISSUE THREAD

**Issue by cat-labs** [danialfarid/ng-file-upload#692]
Hi there,

First of all thanks for the plugin, it's awesome!

Might be a basic question but I try to get the progress of my file upload. It doesn't enter in my progress event. Do I have to add some extra things in php or it should work only with the move_uploaded_file?

My js:
if ($scope.files && $scope.files.length) {
        for (var i = 0; i < $scope.files.length; i++) {
          var file = $scope.files[i];
          $upload.upload({
            url: '/ajax/update_infos',
            fields: {'project': $scope.projectForm},
            file: file
          }).progress(function (evt) {
            var progressPercentage = parseInt(100.0 * evt.loaded / evt.total);
          }).success(function (data, status, headers, config) {
            console.log("updated!");
          });
        }
      }

My php:
$filename = $_FILES['file']['name'];
$destination = $_SERVER['DOCUMENT_ROOT'] . "/upload/test/". $filename;
move_uploaded_file( $_FILES['file']['tmp_name'] , $destination );

**Comment by danialfarid** [comment]
If the success event is being called but not the progress then you might need to modify your server configuration to report the progress event back. That would be part of the server config not in your upload server code.

There is a wiki php page.

**Comment by olso** [comment]
And as a sidenote, try to pick up a framework if you're going to use php

**Comment by digitalkaoz** [comment]
@danialfarid which wiki page? none of them shows how to configure the progress stuff

**Comment by danialfarid** [comment]
You can ask it on stackoverflow and reference it here. It is not particularly related to this plugin but a php question.

**Comment by lsiden** [comment]
The PHP docs and stackoverflow responses regarding this illustrate how to request and display the file upload progress in an iFrame on a new page returned in response to an HTML form-initiated upload.  However, ng-file-upload initiates the upload as an XHR POST.  It is not clear, therefore, what mechanism the web app uses to send the progress responses that the 3rd call-back function argument in Upload.upload() should expect.

In my case, I installed APC, set apc.rfc1867 to '1' (On) and send my payload as thus:

    ------WebKitFormBoundaryE3h0drbZrLx4iowL
    Content-Disposition: form-data; name="APC_UPLOAD_PROGRESS"

    321
    ------WebKitFormBoundaryE3h0drbZrLx4iowL
    Content-Disposition: form-data; name="file[0]"; filename="MyFile.jpg"
    Content-Type: image/jpeg


    ------WebKitFormBoundaryE3h0drbZrLx4iowL--

I assume that the value "321" can be anything unique.  I generated it with new 

    Date().getUTCMilliseconds();

I suppose that I could query the upload status by sending a series of requests that respond by using 

    apc_fetch('upload_<unique-id>');

but then what would be the point of the 3rd callback argument to Upload.upload()?
